precommit.sh

```diff
@@ -8,3 +8,6 @@ docker exec tcf_django node_modules/.bin/eslint --cache --fix -c .config/.eslint
 docker exec tcf_django coverage run manage.py test --keepdb
 # Get code coverage.
 docker exec tcf_django coverage report -m
+# Selenium tests.
+python seleniumtests/browse_selenium_test.py $@
```

Comment:
I tried and tried and tried to get it to work on docker but I just can't lol I'll revisit this later.

Reply:
Basically the issue is if put `docker exec tcf_django` in front, the entire chrome driver fails